Makefile

```diff
@@ -95,6 +95,24 @@ e2e_013: bin/php-scoper.phar
 	$(PHPSCOPER) init --working-dir=build/set013 --no-interaction
 	diff src/scoper.inc.php.tpl build/set013/scoper.inc.php
 
+e2e_014:	## Run end-to-end tests for the fixture set 014: source code case with psr-0
```

Comment:
can you double check that the indent is correct for the lines with `##` when doing `make`?